lint-configs/.clang-format

```
@@ -87,7 +88,10 @@ IntegerLiteralSeparator:
   DecimalMinDigits: 5
   Hex: 4
   HexMinDigits: 4
-KeepEmptyLinesAtTheStartOfBlocks: false
+KeepEmptyLines:
+  AtEndOfFile: true
+  AtStartOfBlock: false
+  AtStartOfFile: false
 LambdaBodyIndentation: "Signature"
 LineEnding: "LF"
 MaxEmptyLinesToKeep: 1
```

```suggestion
MainIncludeChar: "Quote"
MaxEmptyLinesToKeep: 1
```

Another new option omitted from the release notes.